.cloud-repo-tools.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_product": "redis",
+  "id": "redis",
+  "description": "Client for Google Cloud Redis",
```

Comment:
some fields are missing:
```
      "name": "TBD",
      "short_name": "TBD",
      "docs_url": "TBD",
```
which causes weird stuff in `README.md` that are mentioned by @stephenplusplus.